api/source/job_control/request_upload_url.rst

```diff
@@ -38,42 +36,13 @@ URLs have a lifetime of 48 hours.
 
-**Query String Parameters** — Required `(when adding media from URL)`
-
-+------------------+------------------------------------------------------------------------------+
-| Name             | Details                                                                      |
-+==================+==================+===========================================================+
-| media_url        | `Description`    | The URL from which media will be obtained                 |
-|                  +------------------+-----------------------------------------------------------+
-|                  | `Allowed Values` | URL Encoded String                                        |
-|                  +------------------+-----------------------------------------------------------+
-|                  | `Example`        | ``media_url=http%3A%2F%2Fwww.domain.com%2Fvideo.mp4``     |
-+------------------+------------------+-----------------------------------------------------------+
-
 **Request Body** — `(when adding media from local file)`
 
-+------------------+------------------------------------------------------------------------------+
-| Name             | Details                                                                      |
-+==================+==================+===========================================================+
-| not applicable   | `Description`    | Raw binary of a media file                                |
-|                  +------------------+-----------------------------------------------------------+
-|                  | `Allowed Values` | not applicable                                            |
-|                  +------------------+-----------------------------------------------------------+
-|                  | `Example`        | not applicable                                            |
-+------------------+------------------+-----------------------------------------------------------+
+not applicable
 
 **HTTP Headers** — Required `(when uploading LARGE media files)`
 
-+------------------+------------------------------------------------------------------------------+
-| Name             | Details                                                                      |
-+==================+==================+===========================================================+
-| Content-Length   | `Description`    | File size (in bytes)                                      |
-|                  +------------------+-----------------------------------------------------------+
-|                  | `Allowed Values` | Integer                                                   |
-|                  +------------------+-----------------------------------------------------------+
-|                  | `Example`        | ``645809838``                                             |
-+------------------+------------------+-----------------------------------------------------------+
-
+not applicable
 **Responses**
```

Comment:
Create a new line here. 
*not applicable* and **Resposes** are in the same line